source/screens/Home/Transactions/TransactionListItem.tsx

```diff
@@ -1,17 +1,25 @@
 import React, {useMemo} from 'react';
 
 import './TransactionListItem.scss';
+import classnames from 'classnames';
 import {TransactionHistoryItem} from '../../../Background/services/types';
 import moneyFormatter from '../../../utils/moneyFormatter';
 
 type Props = {
   transactionHistoryItem: TransactionHistoryItem;
 };
 
-const TransactionListItem = (props: Props) => {
+const isNegative = (value: string | number): boolean => {
+  if (typeof value === 'string' && value?.length > 0) {
+    return value[0] === '-';
+  }
+  return false;
+};
+
+const TransactionListItem: React.FC<Props> = (props) => {
   const {transactionHistoryItem} = props;
 
-  const {satoshisBalanceImpact, feeImpact} = transactionHistoryItem;
+  const {satoshisBalanceImpact, feeImpact, blockHash} = transactionHistoryItem;
   const balanceImpactFormatted = useMemo(() => {
     const formattedImpact = moneyFormatter.formatDuffs(
       Math.abs(satoshisBalanceImpact + feeImpact)
```

Comment:
Looks like there's a mistake in calculating total balance impact.
Can you please make it `satoshisBalanceImpact - feeImpact` instead?